cmd/evm/internal/t8ntool/execution.go

```diff
@@ -324,8 +322,13 @@ func (pre *Prestate) Apply(vmConfig vm.Config, chainConfig *params.ChainConfig,
 		h := types.DeriveSha(types.Withdrawals(pre.Env.Withdrawals), trie.NewStackTrie(nil))
 		execRs.WithdrawalsRoot = &h
 	}
-	if vmContext.ExcessBlobGas != nil {
-		execRs.CurrentExcessBlobGas = (*math.HexOrDecimal64)(vmContext.ExcessBlobGas)
+	if vmContext.BlobBaseFee != nil {
+		// TODO (MariusVanDerWijden): I have no idea why we have this field at all.
```

Comment:
I think the idea here is that if you provide `parentExcessBlobGas` and `parentBlobGasUsed` t8n will automatically calculate that block's `excessBlobGas`. In these instances you would probably want the calculated value output so that you can pipe it in for the next transition.

Reply:
Fixed, sry

Reply:
I still don't think that this is really needed